vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -14,7 +14,7 @@
         </picture>
         <span class="brand__title">BC Registries <span class="brand__title--wrap">and Online Services</span></span>
       </a>
-      <div class="app-header__actions">
+      <div v-if="!disableActions" class="app-header__actions">
```

Comment:
Double negative is confusing. Easy fix?

Reply:
With a default of true?

Reply:
Hmm, yea lol. 
I'll change the prop title to `showActions` defaulted to `true`